### ISSUE TITLE
Some... responses to CW site design responsivity

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars.com#939]
I happen to browse CW with my mobile phone as well, and here are a list of issues I've noticed with the mobile specific site (with the mobile Chrome browser):

1. Kata/profile page: tabs that are too far to the right of the screen are virtually inaccessible. For example, in landscape mode my kata page doesn't have Forks and Discourse, and the profile page is even worse. Even if I rotated the screen Social and Discourse tabs are out of bounds.

2. Side bar: Leaders button at the bottom is inaccessible, it seem that it's out of the bottom bounds. It only becomes visible if I scroll down and other elements are still scrolling, but even if I click it, it immediately disappears beneath the bottom again.

3. Kata list: search functionality is inaccessible D:

4. Text boxes: there are lots of stuttering when I try to type in things. It seems that if I don't wait for a while between each burst of consecutive inputs, my inputs will be stuttered (as in, word prediction got cut off) per character at some regular interval.

5. Kata solving: Yes, I solve katas with mobile phones. Besides the above-mentioned stuttering issue, frequently inserts a newline for *every character* I typed in. This is horrible.

These problems don't exist while browsing the desktop version of the site on an mobile phone (Chrome browser has settings to make a tab fetch the desktop version instead of mobile version of a site).

So... yeah, at the current state it seems that I should probably just fetch the desktop site on my mobile phone all the time. At least it's more usable I think?

**Comment by tachyonlabs** [comment]
Yeah, I've occasionally tried to do katas on my Android phone or tablet while riding a bus. :-) The only time I was able to get text into the mobile site without duplication and newlines everywhere involved typing my solution into Google Docs and pasting it into the Dolphin browser -- even then there was still some junk following my solution that I couldn't get rid of, but because it came after the return statement my solution still passed.

**Comment by anand-kashyap** [comment]
Issue still persists.
